transplant_guide.md

```diff
@@ -147,39 +147,67 @@ Info about U-district
+  - SOMEONE ELSE NEEDS TO EXPAND THIS! I just use Google Maps :P
+
+**Biking**
+  If you're new to Seattle, here's a warning: the hills are no joke. The thing about the hills is that there aren't a bunch of little hills.... it's that Seattle itself can seem like one GIANT hill. Depending on where you live, it can be a good 30 minute ride uphill the entire way. (It makes getting _to_ Ada easy, though!)
+  That being said, it's not a huge deal as long as you don't have a fixed-gear bike, and as long as you shift down. Don't sprint up the hills -- take your time. And don't feel silly if you get off and walk for a particularly steep block or two -- lots of people do it.
+  See our car-to-bike guide for more information!
+
+**The Ferry**
+  We _strongly_ recommend not living far enough away that you need to take the ferry, simply because commuting takes so much time. That being said, the ferry is a smooth ride, so if you're wondering if you can work while you're on the ferry, the answer is yes.
+
+**The Link**
+  - SOMEONE ELSE?
+
+**Driving**
+  We recommend leaving your car at home and not needing to drive, but if you do.....
+  - KYNA, WANT TO HELP FILL THIS IN?
+
+- SOMEONE ELSE FOR THESE?:
 - transit
-  - ORCALift passes and how to obtain one
     - apps to help you get around
   - bus
   - train
-- car (talk to Kyna)
-- walking
-- ferry (it's an option)
-
 
 Buying a Laptop _Ashley_
 ---------------
-- Student discount 
-- Apple discount 
-- Macbook Pro
-- Macbook Air
-- Macs in General  
-- 10% tax in WA
-- warranties
-  - some credit cards have warranties
+**Student discount**
+  - Note that if you're still in college, you have access to the student discounts. It's worth checking out how much the computers cost on campus, since they sometimes will sell the older versions and will cost a bit less. Additionally, on Apple's website, you can shop with the student discount. Be sure to compare the cost of these to the regularly-priced computers -- sometimes it'll save you money, and sometimes it won't.
+
+**Apple discount**
+If you know someone who works at Apple, you might be able to ask them to buy the computer for you. Be careful to only ask someone if they're very close to you, and be understanding if they say no. You don't want to put anyone in an awkward situation -- some people are not comfortable with using their discounts for friends.
+
+**Macbook Pro vs Macbook Air**
+  - The Air is good enough for what you need in class, but if you plan to use the computer forever, it can be nice to get the Pro. These tend to have more RAM, and let you upgrade the RAM even further, so they won't seem as slow in a couple of years. That being said, it's highly recommended to upgrade the RAM if you can afford it (again, this won't matter as much in Ada but you'll appreciate it down the road). Do what makes the most sense for your budget.
+
+- ANYONE ELSE HAVE ANYTHING TO ADD ON THESE?
+  - Macbook Pro
+  - Macbook Air
+  - Macs in General
+  - 10% tax in WA
```

Comment:
Looks like we are at 9.6% because we're part of the transportation benefit district, which I think is Sound Transit's mechanism of funding. http://dor.wa.gov/Docs/forms/ExcsTx/LocSalUseTx/LocalSlsUseFlyer_15_Q3_alpha.pdf